resync/api/middleware/cors_monitoring.py

```
@@ -201,77 +129,38 @@ def monitor_cors() -> CORSMonitor:
     """
     return CORSMonitor()
 
-
 class CORSMonitoringMiddleware:
     """
     Middleware to monitor and log CORS-related activities.
     """
 
-    def __init__(self, app, allowed_origins: list[str] = None):
+    def __init__(self, app, allowed_origins: list[str]=None):
```

Ruff (0.15.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)